static/app/views/alerts/rules/issue/details/ruleDetails.spec.tsx

```diff
@@ -252,6 +252,34 @@ describe('AlertRuleDetails', () => {
     ).not.toBeInTheDocument();
   });
 
+  it('disabled rule can be re-enabled', async () => {
+    const disabledRule = TestStubs.ProjectAlertRule({
+      status: 'disabled',
+      disableDate: moment().subtract(1, 'day').format(),
+      disableReason: 'noisy',
+    });
+    MockApiClient.addMockResponse({
+      url: `/projects/${organization.slug}/${project.slug}/rules/${disabledRule.id}/`,
+      body: disabledRule,
+      match: [MockApiClient.matchQuery({expand: 'lastTriggered'})],
+    });
+    const enableMock = MockApiClient.addMockResponse({
+      url: `/projects/${organization.slug}/${project.slug}/rules/${disabledRule.id}/enable/`,
+      method: 'PUT',
+    });
+    createWrapper();
+    expect(
+      await screen.findByText(/This alert was disabled due to lack of activity/)
+    ).toBeInTheDocument();
+
+    await userEvent.click(screen.getByRole('button', {name: 'Click Here'}));
+
+    expect(enableMock).toHaveBeenCalled();
+    expect(
+      screen.queryByText(/This alert is scheduled to be disabled/)
```

Comment:
Should this be checking that "this alert was disabled due to lack of activity" is not shown instead? or both this and that aren't shown?